src/admin.rs

```diff
@@ -194,11 +222,13 @@ impl FluvioAdmin {
                             .collect::<Array>(),
                     )
                 })
-                .map_err(|e| FluvioError::from(e).into());
-            topic_list
-        })
+                .map_err(|e| FluvioError::from(e).into())
+        });
+        // WARNING: this does not validate the return type. Check carefully.
+        promise.unchecked_into()
```

Comment:
I know it's not fully needed here with the function return type but I like the [turbofish](https://turbo.fish/) as the [`unchecked_into` can panic at runtime](https://docs.rs/wasm-bindgen/0.2.79/wasm_bindgen/trait.JsCast.html#method.unchecked_into) so it would at least be clearer for future maintenance.

I wish one could make custom lint for this type of thing.